.coafile

```diff
@@ -0,0 +1,11 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js
```

Comment:
ignore `.git` also?

Reply:
locally coala will run on .git also

Reply:
Coala shouldn't check `.git`. If it does, then it is a bug.